src/Features/CSharp/Portable/PullMemberUp/CSharpMembersPullerService.cs

```diff
@@ -27,60 +26,31 @@ public CSharpMembersPullerService()
         {
         }
 
-        protected override SyntaxTriviaList GetFirstMemberTrivia(SyntaxNode root)
+        protected override SyntaxList<SyntaxNode> GetMembers(SyntaxNode root)
         {
-            var members = root switch
+            return root switch
             {
                 CompilationUnitSyntax compilationUnit => compilationUnit.Members,
                 NamespaceDeclarationSyntax namespaceDeclaration => namespaceDeclaration.Members,
                 _ => throw ExceptionUtilities.UnexpectedValue(root)
             };
```

Comment:
this method is unneeded.  the base type can use ISyntaxFactsService and members like:

```
        SyntaxList<SyntaxNode> GetMembersOfNamespaceDeclaration(SyntaxNode namespaceDeclaration);
        SyntaxList<SyntaxNode> GetMembersOfCompilationUnit(SyntaxNode compilationUnit);
```

to do this work.